### PR TITLE
feat(useMark): new function

### PR DESCRIPTION
**Pull request by Jclong98**
### Description

New `@vueuse/integrations` function for [`mark.js`](https://markjs.io/)

### Additional context

I was unsure how to get the type for `Mark` so I made a small mock for the parts that were used in [`useMark`](https://github.com/Jclong98/vueuse/blob/main/packages/integrations/useMark/index.ts#L17). Is this acceptable practice?

---

### What is the purpose of this pull request? 

- [ ] Bug fix
- [x] New Feature
- [ ] Documentation update
- [ ] Other

### Before submitting the PR, please make sure you do the following

- [x] Read the [Contributing Guidelines](https://github.com/vueuse/vueuse/blob/main/CONTRIBUTING.md).
- [x] Read the [Pull Request Guidelines](https://github.com/vueuse/vueuse/blob/main/packages/guidelines.md).
- [x] Check that there isn't already a PR that solves the problem the same way to avoid creating a duplicate.
- [x] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
- [ ] Ideally, include relevant tests that fail without this PR but pass with it.

### CODE REVIEW

File: packages/integrations/useMark/index.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import { computed, ref } from 'vue-demi'
+import type { Ref } from 'vue-demi'
```
Comment (songjianet) [inline packages/integrations/useMark/index.ts:2]: import type is best placed at the bottom of all references.

**Review by songjianet — CHANGES_REQUESTED** [review]
Can be used as a reference.

File: packages/integrations/useMark/index.ts (hunk 1 above)
Comment (Jclong98) [reply packages/integrations/useMark/index.ts]: I've moved the type imports under the rest of the imports, is that what you meant?

**Comment by antfu** [comment]
Thank you. Unfortunately we no longer accept new functions. Feel free to publish a standalone package on your own.

**Pull request closed without merging**